src/main/webapp/index.html

```diff
@@ -27,7 +28,7 @@
   <!-- Navigation -->
   <nav class="navbar navbar-expand-lg navbar-dark navbar-custom fixed-top">
     <div class="container">
-      <a class="navbar-brand" href="#">Techish</a>
+      <img class="techish-logo" src="img/techish_logo.png" alt="">
```

Comment:
nit [L31]: Since you added 'alt', might as well give it a descriptive value. Alt tags are used by screen readers / accessibility / when a resource fails to display. It's encouraged on all main elements, mainly due to accessibility concerns.

Reply:
Done